### PR TITLE
Add basic react native app

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install npm dependencies
+        run: |
+          npm install
+      
+      - name: Run link
```
Comment (valfirst) [inline .github/workflows/ci.yml:21]: `link` -> `lint`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install npm dependencies
+        run: |
+          npm install
```
Comment (ikalinin1) [inline .github/workflows/ci.yml:19]: add super lineter? :)

File: .buckconfig [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
[the remark this answers is not in the document]
Comment (uarlouski) [reply .buckconfig:1]: I left this file untouched after project generation, probably there is some hidden intent :)

File: .github/workflows/ci.yml (hunk 2 above)
Comment (uarlouski) [reply .github/workflows/ci.yml:19]: It's worth to try in the future :)

File: .buckconfig [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+[android]
+  target = Google Inc.:Google APIs:23
```
Comment (ikalinin1) [inline .buckconfig:3]: why the target is 23 which is android 6 ?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ikalinin1) [reply .github/workflows/ci.yml:19]: imho js based app is a best choice to use it asap

File: __tests__/App-test.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../src/App';
+
+// Note: test renderer must be required after react-native.
```
Comment (ikalinin1) [inline __tests__/App-test.js:9]: :O

File: app.json [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "vividusTestApp",
+  "displayName": "vividusTestApp"
```
Comment (ikalinin1) [inline app.json:3]: Could it be `Human readable? :) Vividus Test App

File: .buckconfig (hunk 4 above)
Comment (uarlouski) [reply .buckconfig:3]: it's default value, actual value will be defined based on the latest supported version of android provided by bs/saucelabs. Will be changed later

File: __tests__/App-test.js (hunk 5 above)
Comment (uarlouski) [reply __tests__/App-test.js:9]: default stuff

File: .github/workflows/ci.yml (hunk 2 above)
Comment (uarlouski) [reply .github/workflows/ci.yml:19]: I meant files like json etc, superlinter already uses eslint for js based apps

**Pull request merged**